app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,370 @@
+
+        <TextView
+            android:id="@+id/countReviews_textView"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="9dp"
+            android:layout_marginTop="4dp"
+            android:fontFamily="@font/roboto"
+            android:text="@string/Reviews"
+            android:textColor="@color/fade"
+            android:textSize="14sp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/imageView"
+            app:layout_constraintTop_toBottomOf="@+id/tagTextView" />
+
+        <TextView
+            android:id="@+id/storyline_name_textView"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="24dp"
+            android:layout_marginEnd="16dp"
+            android:text="@string/Stroryline_name"
+            android:textColor="@color/ECECEC"
+            android:textSize="14sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/countReviews_textView" />
+
+        <TextView
+            android:id="@+id/storyline_text_textView"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="4dp"
+            android:layout_marginEnd="16dp"
+            android:text="@string/Storyline_text"
+            android:textAlignment="textStart"
+            android:textColor="@color/white"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/storyline_name_textView" />
+
+        <TextView
+            android:id="@+id/casts_textView"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="24dp"
+            android:layout_marginEnd="341dp"
+            android:text="@string/Casts"
+            android:textAlignment="textStart"
+            android:textColor="@color/ECECEC"
+            android:textSize="14sp"
+            android:textStyle="normal|bold"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/storyline_text_textView" />
+
+        <ImageView
+            android:id="@+id/actor_imageView"
+            android:layout_width="80dp"
+            android:layout_height="80dp"
+            android:layout_marginStart="16dp"
+            android:layout_marginTop="7dp"
+            android:contentDescription="@string/actor_name"
```

Comment:
remove
android:contentDescription="@string/actor_name"
android:visibility="visible"
tools:srcCompat="@drawable/downey"

here and in images below